test/integration/targets/loops/test_loop_item_display.yml

```
@@ -2,8 +2,8 @@
 - hosts: localhost
   gather_facts: no
   tasks:
-    - name: Sleep 5 seconds per loop item
-      shell: echo {{ item }} && sleep 5
+    - name: Sleep 2 seconds per loop item
+      shell: echo {{ item }} && sleep 2
       loop:
         - 1
```

No reason to sleep on the first one at all, so long as there's a measurable gap between the appearance of the first one and the second (which the test script already enforces and the 0-2s should :crossed_fingers: be plenty long for any modern CI host).
```suggestion
   - name: Sleep on second loop item
      shell: echo {{ item }} && sleep {{ item }}
      loop:
        - 0
```

---

Okay, the margin has been reduced. I was concerned sleeping 3 seconds for each in the script could cause a race condition if the second loop completes too quickly, but it hasn't for me yet.